node/src/accountant/payment_adjuster/non_unit_tests/mod.rs

```diff
@@ -0,0 +1,1258 @@
+        error
+    ))
+    .unwrap()
+}
+
+fn resolve_affordable_transaction_count(adjustment: &Adjustment) -> String {
+    match adjustment {
+        Adjustment::ByServiceFee => "UNLIMITED".to_string(),
+        Adjustment::TransactionFeeInPriority {
+            affordable_transaction_count,
+        } => affordable_transaction_count.to_string(),
+    }
+}
+
+fn write_thick_dividing_line(file: &mut dyn Write) {
+    write_ln_made_of(file, '=')
+}
+
+fn write_thin_dividing_line(file: &mut dyn Write) {
+    write_ln_made_of(file, '_')
+}
+
+const PAGE_WIDTH: usize = 120;
+
+fn write_ln_made_of(file: &mut dyn Write, char: char) {
+    let _ = file
+        .write_fmt(format_args!("{}\n", char.to_string().repeat(PAGE_WIDTH)))
+        .unwrap();
+}
+
+fn prepare_interpretable_adjustment_result(
+    account_info: AccountInfo,
+    resulted_affordable_accounts: &mut Vec<PayableAccount>,
+) -> InterpretableAdjustmentResult {
+    let adjusted_account_idx_opt = resulted_affordable_accounts
+        .iter()
+        .position(|account| account.wallet == account_info.wallet);
+    let bills_coverage_in_percentage_opt = match adjusted_account_idx_opt {
+        Some(idx) => {
+            let adjusted_account = resulted_affordable_accounts.remove(idx);
+            assert_eq!(adjusted_account.wallet, account_info.wallet);
+            let bill_coverage_in_percentage = {
+                let percentage = (adjusted_account.balance_wei * 100)
+                    / account_info.initially_requested_service_fee_minor;
+                u8::try_from(percentage).unwrap()
+            };
+            Some(bill_coverage_in_percentage)
+        }
+        None => None,
+    };
+    InterpretableAdjustmentResult {
+        info: AccountInfo {
+            wallet: account_info.wallet,
+            debt_age_s: account_info.debt_age_s,
+            initially_requested_service_fee_minor: account_info
+                .initially_requested_service_fee_minor,
+        },
+
+        bills_coverage_in_percentage_opt,
+    }
+}
+
+fn sort_interpretable_adjustments(
+    interpretable_adjustments: Vec<InterpretableAdjustmentResult>,
+) -> (Vec<InterpretableAdjustmentResult>, bool) {
+    let (finished, eliminated): (
+        Vec<InterpretableAdjustmentResult>,
+        Vec<InterpretableAdjustmentResult>,
+    ) = interpretable_adjustments
+        .into_iter()
+        .partition(|adjustment| adjustment.bills_coverage_in_percentage_opt.is_some());
+    let were_no_accounts_eliminated = eliminated.is_empty();
+    let finished_sorted = finished.into_iter().sorted_by(|result_a, result_b| {
+        Ord::cmp(
+            &(
+                result_b.bills_coverage_in_percentage_opt,
+                result_a.info.initially_requested_service_fee_minor,
+            ),
+            &(
+                result_a.bills_coverage_in_percentage_opt,
+                result_b.info.initially_requested_service_fee_minor,
+            ),
+        )
+    });
```

Comment:
It's hard to read. Please add the comment explaining that:

The priority order will be such that - Percentage Descending, Service Fee Ascending